src/Ozds.Client/Shared/MeterGraph.razor

```diff
@@ -28,48 +28,50 @@
                                YValue="x => 0"
                                Stroke="@(ColorGraph(0))"/>
             }
-          </ApexChart>
-          <MudItem Class="d-flex flex-column justify-center gap-2">
-            <MudMenu Dense="true" Variant="Variant.Filled" Color="Color.Primary">
-              <ActivatorContent>
-                <MudButton Variant="Variant.Filled" Color="Color.Primary">
-                  @_dataTitle
-                  <MudIcon Icon="@Icons.Material.Filled.KeyboardArrowDown"/>
-                </MudButton>
-              </ActivatorContent>
-              <ChildContent>
-                @foreach (var type in _displayType)
-                {
-                  <MudMenuItem OnClick="@(() => DataMenuItemClicked(type))">@type</MudMenuItem>
-                }
-              </ChildContent>
-            </MudMenu>
-            <MudMenu Dense="true" Variant="Variant.Filled" Color="Color.Primary">
-              <ActivatorContent>
-                <MudButton Variant="Variant.Filled" Color="Color.Primary">
-                  @_timeTitle
-                  <MudIcon Icon="@Icons.Material.Filled.KeyboardArrowDown"/>
-                </MudButton>
-              </ActivatorContent>
-              <ChildContent>
-                <MudMenuItem OnClick="@(() => TimeMenuItemClicked("Scale1"))">15 min</MudMenuItem>
-                <MudMenuItem OnClick="@(() => TimeMenuItemClicked("Scale2"))">1 h</MudMenuItem>
-                <MudMenuItem OnClick="@(() => TimeMenuItemClicked("Scale3"))">6 h</MudMenuItem>
-                <MudMenuItem OnClick="@(() => TimeMenuItemClicked("Scale4"))">24 h</MudMenuItem>
-              </ChildContent>
-            </MudMenu>
-          </MudItem>
-        }
+
+            <MudItem Class="d-flex flex-column justify-center gap-2">
+              <MudMenu Dense="true" Variant="Variant.Filled" Color="Color.Primary">
+                <ActivatorContent>
+                  <MudButton Variant="Variant.Filled" Color="Color.Primary">
+                    @_dataTitle
+                    <MudIcon Icon="@Icons.Material.Filled.KeyboardArrowDown"/>
+                  </MudButton>
+                </ActivatorContent>
+                <ChildContent>
+                  @foreach (var type in _displayType)
+                  {
+                    <MudMenuItem OnClick="@(() => DataMenuItemClicked(type))">@type</MudMenuItem>
+                  }
+                </ChildContent>
+              </MudMenu>
+              <MudMenu Dense="true" Variant="Variant.Filled" Color="Color.Primary">
+                <ActivatorContent>
+                  <MudButton Variant="Variant.Filled" Color="Color.Primary">
+                    @_timeTitle
+                    <MudIcon Icon="@Icons.Material.Filled.KeyboardArrowDown"/>
+                  </MudButton>
+                </ActivatorContent>
+                <ChildContent>
+                  <MudMenuItem OnClick="@(() => TimeMenuItemClicked("Scale1"))">15 min</MudMenuItem>
```

Comment:
na ovakvim stvarima zas ne stavis TimeSpan.FromMinutes() ili TimeSpan.FromHours() u argument za TimeMenuItemClicked?
mislim to ce ti i smanjit kod kada dolje trebas to u query prenjet

Reply:
jer mi je ovak jednostavnije i netrebam baratat sa timespan-ovima , dal to trebam izmjenit ?

Reply:
nazovi na teams kad mozes pa ti pokazem kak mozes jos jednostavnije